src/widget/wcolorpicker.cpp

```diff
@@ -59,6 +64,8 @@ WColorPicker::WColorPicker(QWidget* parent)
             row++;
         }
 
+        connect(this, &WColorPicker::colorPicked,
```

Comment:
Splitting arguments on separate lines may improve readability, especially for the lambda argument.